src/Manipulators/BaseManipulator.php

```diff
@@ -37,6 +37,13 @@ public function getParam(string $name): mixed
             : null;
     }
 
+    /**
+     * Get the names of the manipulator API parameters.
+     *
+     * @return list<string>
+     */
+    abstract public function getApiParams(): array;
```

Comment:
We don't need this abstract method because it is defined already in ManipulatorInterface

Reply:
I agree, I was following what was done for the `run` method, I removed both from `BaseManipulator`?